iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/functionclass/EventDataListener.java

```diff
@@ -220,13 +231,11 @@ public EventDataListener(final Device device, final Scheduler scheduler, final D
                     case "tOnT": {
 
                         logger.info("Time On Type found.");
-                        this.device.getRelay(relayNr)
-                                .getSchedule(scheduleNr)
+                        this.device.getRelay(relayNr).getSchedule(scheduleNr)
```

Comment:
You can use the schedulesToUpdate variable here as well. You can use it in all places, where you have "this.device.getRelay(relayNr).getSchedule(scheduleNr)".

Reply:
Done